package.json

```diff
@@ -79,6 +79,11 @@
             "type": "boolean",
             "default": true,
             "description": "Whether decoded text should be shown in the editor."
+          },
+          "hexeditor.showOpenFileButton": {
+            "type": "boolean",
+            "default": true,
```

Comment:
Please default this to false as I understand users who want it should have the ability to have it, but in general I don't want to clutter everyone's toolbar

Reply:
updated